src/snovault/helpers/helper.py

```diff
@@ -172,8 +172,8 @@ def get_search_fields(request, doc_types):
     types = request.registry[TYPES]
     for doc_type in doc_types:
         type_info = types[doc_type]
-        for value in type_info.schema.get('boost_values', ()):
-            fields.add('embedded.' + value)
+        for value, boost in type_info.schema.get('boost_values', {}).items():
+            fields.add('embedded.' + value + '^' + str(boost))
```

Comment:
so this theoretically makes the boost values > 1.0 actually do something?  I thought we were not going to do index-time boosting anyway...   Still it seems harmless.

Reply:
This is query-time boosting. The fields get added to the query_string query like this example from ES docs: 
```
curl -X GET "localhost:9200/_search" -H 'Content-Type: application/json' -d'
{
    "query": {
        "query_string" : {
            "fields" : ["content", "name^5"],
            "query" : "this AND that OR thus",
        }
    }
}
```
Since mostly all of these fields are keyword fields the boost comes into play when there is an exact match. This helps when searching for accession for example. The actual item with that accession gets ranked higher. Also when searching for biosample age, which otherwise can get lost in results that match on more fields but have the wrong age.